server/services/group.js

```diff
@@ -115,6 +115,12 @@ function addAdmin(groupName, userId) {
     .then(validatedUserId => Group.addAdmin(groupName, validatedUserId));
 }
 
+function removeAdmin(groupName, userId) {
+  return userService.exists(userId)
```

Comment:
Should there be some kind of logic to ensure that the group isn't left without an admin? Ie, you can't remove an admin if it'll leave the array empty?